### PR TITLE
Upgraded `@ckeditor/ckeditor5-dev-utils` to the latest version.

### PR DESCRIPTION
**Pull request by pomek**
Fixes: #31.

Requires: https://github.com/ckeditor/ckeditor5-dev-utils/pull/33.

### CODE REVIEW

File: lib/tasks/create-package/index.js [hunk 1]
```diff
@@ -84,13 +85,13 @@ module.exports = ( ckeditor5Path, workspaceRoot ) => {
 				'{{ProjectDescription}}': packageDescription
 			};
 
-			log.out( `Initializing repository ${ repositoryPath }...` );
+			log.info( `Initializing repository ${ repositoryPath }...`, { raw: true } );
```
Comment (Reinmar) [inline lib/tasks/create-package/index.js]: It doesn't need to be raw now. It is a normal message AFAICS and could be logged normally.

Comment (Reinmar) [inline lib/tasks/create-package/index.js]: This applies to most of the changes in this PR.

**Comment by Reinmar** [comment]
There's something wrong (this is a result of `gulp st`):

image

For comparison, this is before the changes:

image

I haven't tested other methods.

**Comment by pomek** [comment]
In a previous version of `@ckeditor/ckeditor5-dev-utils`, method `git.status` worked in silent mode: https://github.com/ckeditor/ckeditor5-dev-utils/blob/v1.1.0/lib/git.js#L128. Now, API of Utils is incompatible with itself.

Naturally, I will change a verbosity level to `error`.

**Comment by Reinmar** [comment]
Yyyy... I don't understand. How was it silent if it printed out some results? Now it prints them twice, which is a bit weird. Could you explain that?

**Comment by pomek** [comment]
Let's go through the code, line by line:

1. `gulp status` call the [`ckeditor5-dev-env/lib/tasks/status`](https://github.com/ckeditor/ckeditor5-dev-env/blob/master/lib/tasks/status/index.js#L20) module.
2. Each package is calling the [`ckeditor5-dev-utils/git.getStatus`](https://github.com/ckeditor/ckeditor5-dev-utils/blob/v2.1.0/lib/git.js#L128) method.
3. `git.getStatus` is calling the [`tools.shExec`](https://github.com/ckeditor/ckeditor5-dev-utils/blob/v2.1.0/lib/tools.js#L22) method from the same package. At this moment `shExec` receives invalid parameters.
4. `tools.shExec` writes an output from called command and [returns the output as string](https://github.com/ckeditor/ckeditor5-dev-utils/blob/v2.1.0/lib/tools.js#L45).
5. The output is backing to `status` module from `ckeditor5-dev-env` where [it is displayed again](https://github.com/ckeditor/ckeditor5-dev-env/blob/master/lib/tasks/status/index.js#L38).

Do you understand now? :D

**Comment by pomek** [comment]
I've updated the description. This PR needs https://github.com/ckeditor/ckeditor5-dev-utils/pull/33.

**Comment by Reinmar** [comment]
I released new version of dev-utils, but there's the same problem with `gulp update`:

image

**Comment by Reinmar** [comment]
Does `git` print these things to stderr that they are red? They are normal messages...

**Comment by pomek** [comment]
> Does git print these things to stderr that they are red?

Yes. I don't know why but Git returns string below:

> From github.com:ckeditor/ckeditor5-enter
> * branch            master     -> FETCH_HEAD

to `stderr`.

image

**Comment by pomek** [comment]
I guess `tools.shExec` must be a little smarter. At this moment this function logs `stdout` as `log.info` and `stderr` as `log.error`. 

We should rather base on `ret.code`. If `ret.code === 0` then both outputs should be logged by `log.info`. In the other case - use `log.error`.

**Comment by pomek** [comment]
Ok. Since https://github.com/ckeditor/ckeditor5-dev-utils/issues/34 is closed, you can review it again.

**Pull request merged**